jsundet-HartfordCT.md

```diff
@@ -0,0 +1,6 @@
+# HartfordCT
+### places to visit
```

Comment:
looks none one is interested in reviewing my changes. and workflow doesnt allow me review me my own changes :(.

could you please do me a favour reviewing my changes.